astor/rtrip.py

```diff
@@ -160,7 +160,7 @@ def usage(msg):
 
         Usage:
 
-            python -m astor.rtrip [readonly] [<source>]
+            python3 -m astor.rtrip [readonly] [<source>]
```

Comment:
astor still supports Python 2.7 (it will be dropped in either 0.9 or 1.0) and `python` may still point to Python 3 especially if astor is installed inside a virtual environment. I'd prefer keeping it as for now.

It would be nice to use argparse instead of the current solution, though.